parser/src/parser/tests/integrity_constraints.rs

```diff
@@ -232,7 +232,8 @@ fn err_missing_integrity_constraint() {
         let b = [a, 2 * a]
         let c = [[a - 1, a^2], [b[0], b[1]]]";
     let error = Error::ParseError(ParseError::MissingIntegrityConstraint(
-        "Declaration of at least one integrity constraint is required".to_string(),
+        "Declaration of at least one integrity constraint or evaluator function call is required"
+            .to_string(),
```

Comment:
I probably wouldn't change this. In my mind, an evaluator function call is just a special type of constraint. It's still prefixed by the `enf` keyword that we use for constraints, so I don't think the user should need to think about it as a separate thing.

Reply:
Makes sense. Thanks.